src/i18n/en/common.json

```diff
@@ -2,6 +2,37 @@
   "error": {
     "pageNotFound": "404 Page not found"
   },
+  "favouritesPage": {
+    "notification": {
+      "body": "You can find all the selected targets here. You can return to plot search page to add more targets.",
+      "link": "Return to plot search page",
+      "title": "Do you want to search for more targets?"
+    },
+    "targetCard": {
+      "removeButton": "",
```

Comment:
This was missed in the review of the last pull request, but the default translation for this message is missing from the relevant source code file, so the translation extractor emptied it here as well.